src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -44,93 +51,97 @@
   }
 
   /**
-   * Prompt the user for a valid password
+   * Prompt the user for a valid password.
```

Comment:
You might consider making sure all the verb forms end in `-s` throughout the javadoc in this PR since you're making changes to the class as a whole. 
```suggestion
   * Prompts the user for a valid password.
```